neqo-qpack/src/encoder.rs

```diff
@@ -14,21 +14,36 @@ use crate::stats::Stats;
 use crate::table::{HeaderTable, LookupResult, ADDITIONAL_TABLE_ENTRY_SIZE};
 use crate::Header;
 use crate::{Error, QpackSettings, Res};
-use neqo_common::{qdebug, qlog::NeqoQlog, qtrace};
+use neqo_common::{qdebug, qerror, qlog::NeqoQlog, qtrace};
 use neqo_transport::Connection;
 use std::collections::{HashMap, HashSet, VecDeque};
 use std::convert::TryFrom;
 
 pub const QPACK_UNI_STREAM_TYPE_ENCODER: u64 = 0x2;
 
+#[derive(Debug, PartialEq)]
+enum LocalStreamState {
+    NoStream,
+    Uninitialized(u64),
+    Initialized(u64),
```

Comment:
If these are stream IDs, can you use `StreamId`.  If for no other reason than we have far too many u64s floating around.

Reply:
I will switch it to StreamId here.